.github/workflows/main.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
```

Comment:
Could we give this a more descriptive name?

Reply:
Fixed in [d5e81a1](https://github.com/scientificcomputing/generate-python-package/pull/15/commits/d5e81a1e8310c6de3959f127131751096824ba0e)